client/signup/steps/site-topic/index.jsx

```diff
@@ -171,25 +93,14 @@ const mapDispatchToProps = ( dispatch, ownProps ) => ( {
 
 		goToNextStep( flowName );
 	},
-
-	setSiteVertical: verticalData => dispatch( setSiteVertical( verticalData ) ),
 } );
 
 export default localize(
```

Comment:
Here `localize` is not necessary anymore. Can you remove it? :)

Reply:
Nice find 😉 Removed in https://github.com/Automattic/wp-calypso/pull/30730/commits/4fb976a887b36e13d5771318e5d3cffbb52c9a04